artifacts_test.py

```diff
@@ -248,6 +248,9 @@ def verify_xfs_online_discard_enabled(self) -> None:
         mount_options = run("findmnt -no options -t xfs -T /var/lib/scylla").stdout.strip().split(",")
         self.assertIn("discard", mount_options)
 
+        if self.node.is_ubuntu20():
```

Comment:
what's the point of a test, if we disable it ourself, it would mask the whole point of testing our own images.

i think we need to do the fstrim check on for preinstalled scylla

Reply:
in artifacts test we don't run on a pre-installed Scylla, we run in a pure OS image

Reply:
ami test is also artifact test. (the same one)

Reply:
didn't see it failing, and for Ubuntu20 it is stopping the whole next promotion... the AMI should use what OS?